rocketmq-connect/rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/tools/MQConnectAdminStartup.java

```diff
@@ -0,0 +1,115 @@
+package org.apache.rocketmq.connect.runtime.tools;
+
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.PosixParser;
+import org.apache.rocketmq.common.MQVersion;
+import org.apache.rocketmq.connect.runtime.tools.command.*;
+import org.apache.rocketmq.connect.runtime.utils.ServerUtil;
+import org.apache.rocketmq.remoting.protocol.RemotingCommand;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MQConnectAdminStartup {
+    protected static List<SubCommand> subCommandList = new ArrayList<SubCommand>();
+
+
+    public static void main(String[] args) {
```

Comment:
how about lets add a readme for this CLI tool ~

Reply:
Added README